netstandard/ref/System.Security.Cryptography.X509Certificates.cs

```diff
@@ -47,6 +72,16 @@ public enum StoreName
         TrustedPeople = 7,
         TrustedPublisher = 8,
     }
+    public sealed partial class SubjectAlternativeNameBuilder
+    {
+        public SubjectAlternativeNameBuilder() { }
+        public void AddDnsName(string dnsName) { }
+        public void AddEmailAddress(string emailAddress) { }
+        public void AddIpAddress(System.Net.IPAddress ipAddress) { }
+        public void AddUri(System.Uri uri) { }
+        public void AddUserPrincipalName(string upn) { }
+        public System.Security.Cryptography.X509Certificates.X509Extension Build(bool critical = false) { throw null; }
```

Comment:
I'm surprised to see an optional parameter here. (I've verified it's optional in the framework too.)

I thought the system framework design guidelines prohibited this? Maybe I'm years behind the trend.
Making this a required parameter would break source compatibility, but preserve binary compatibility.

The source compatibility would be easy for users to fix when changing their code to target .NET Standard, in the same way as anyone being broken by parameter names being changed.

I think we should consider whether we want this - and a general policy on optional parameters. (If we don't already have one. If we do, please enlighten me :)